ImperatorToCK3/Data_Files/configurables/culture_map.txt

```diff
@@ -197,7 +185,7 @@ link = { ck3 = saxon imp = saxonian imp = anglian imp = gothonic imp = raumarici
 link = { ck3 = bavarian imp = saxonian imp = anglian imp = gothonic imp = raumarician imp = suiones imp = reudingian imp = cimbrian imp = rugian imp = herulian imp = ingvaeonic imp = istvaeonic imp = teutonian imp = gutones imp = lemovian imp = bastarnae imp = irminonic imp = helveconian imp = suebian imp = frisian imp = vandal ck3Province = 2964 ck3Province = 2945 ck3Province = 2949 ck3Province = 2942 ck3Province = 2957 ck3Province = 2959 ck3Province = 2956 ck3Province = 2883 ck3Province = 2960 ck3Province = 2878 ck3Province = 2990 ck3Province = 2987 ck3Province = 2989 ck3Province = 2975 ck3Province = 2977 ck3Province = 3135 ck3Province = 3131 ck3Province = 2971 ck3Province = 3117 ck3Province = 3092 ck3Province = 3109 ck3Province = 3121 ck3Province = 3086 ck3Province = 3081 ck3Province = 3089 ck3Province = 3073 ck3Province = 3093 ck3Province = 3088 ck3Province = 2950 ck3Province = 2955 ck3Province = 3134 ck3Province = 3105 ck3Province = 3128 ck3Province = 3125 ck3Province = 3099 }
-link = { ck3 = frisian ck3Region = custom_netherlands imp = saxonian imp = anglian imp = gothonic imp = raumarician imp = suiones imp = reudingian imp = cimbrian imp = rugian imp = herulian imp = ingvaeonic imp = istvaeonic imp = teutonian imp = gutones imp = lemovian imp = bastarnae imp = irminonic imp = helveconian imp = suebian imp = frisian imp = vandal ck3Province = 2115 ck3Province = 2125 ck3Province = 2105 ck3Province = 2106 ck3Province = 2406 ck3Province = 2405 ck3Province = 2112 ck3Province = 2450 ck3Province = 2428 ck3Province = 2429 ck3Province = 2424 ck3Province = 2421 ck3Province = 2417 ck3Province = 2414 ck3Province = 2418 ck3Province = 2413 }
+link = { ck3 = frisian region = custom_netherlands imp = saxonian imp = anglian imp = gothonic imp = raumarician imp = suiones imp = reudingian imp = cimbrian imp = rugian imp = herulian imp = ingvaeonic imp = istvaeonic imp = teutonian imp = gutones imp = lemovian imp = bastarnae imp = irminonic imp = helveconian imp = suebian imp = frisian imp = vandal ck3Province = 2115 ck3Province = 2125 ck3Province = 2105 ck3Province = 2106 ck3Province = 2406 ck3Province = 2405 ck3Province = 2112 ck3Province = 2450 ck3Province = 2428 ck3Province = 2429 ck3Province = 2424 ck3Province = 2421 ck3Province = 2417 ck3Province = 2414 ck3Province = 2418 ck3Province = 2413 }
```

Comment:
`region` is not a valid keyword, replace it with `ck3Region`

Reply:
Done